css/layout.css

```diff
@@ -50,6 +50,11 @@
   position: absolute;
 }
 
+#play-again {
+  float: right;
```

Comment:
Why `float`? I would use `text-align: right` on a div holding this button, personally